api/core/Directus/Slim/ResponseCacheMiddleware.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Directus\Slim;
+
+use Directus\Bootstrap;
+use Directus\Cache\Response;
+use Slim\Middleware;
+
+class ResponseCacheMiddleware extends Middleware
```

Comment:
Isn't this Middleware working similar to `HttpCacheMiddleware` (Previous `CacheMiddleware`)?